lib/pry/platform.rb

```diff
@@ -0,0 +1,93 @@
+module Pry::Platform
+  extend self
```

Comment:
is this equivalent to `module_function` ?

Reply:
not exactly, the methods aren't registered as private, but i don't think that's an issue? For `Pry#h` it was a hassle.